dotcom-rendering/src/web/layouts/LiveLayout.tsx

```diff
@@ -218,20 +209,22 @@ const sticky = css`
 	}
 `;
 
-const keyEventsTopMarginDesktop = css`
+const keyEventsTopMargin = css`
 	${from.desktop} {
 		margin-top: ${space[1]}px;
 	}
 `;
 
-const stretchLines = css`
```

Comment:
How did you get around the need for negative margins here?

Reply:
We no longer need to stretch the lines as they no longer go full width below desktop or above:

Here's a screenshot of the design on desktop
<img width="384" alt="Screen Shot 2021-12-01 at 19 15 03" src="https://user-images.githubusercontent.com/16781258/144196964-4ef62157-975d-46bb-9b47-7d7d74ed2901.png">

And here it is tablet and below
<img width="545" alt="Screen Shot 2021-12-01 at 19 15 33" src="https://user-images.githubusercontent.com/16781258/144197050-b177bcf5-098e-40ea-b93c-0ccf3039d751.png">